core/src/sdk/README.md

```diff
@@ -0,0 +1,103 @@
+### Build C++ sdk
+
+The C++ sdk source code is under milvus/core/src/sdk. Build entire milvus project will also build the sdk project. 
+If you don't want to build entire milvus project, you can do the following steps:
+```shell
+ # generate make files
+ $ cd [Milvus root path]/core
+ $ ./build.sh -l
+ 
+ # build C++ sdk project
+ $ cd [Milvus root path]/core/cmake_build
+ $ make -C src/sdk
+```
+
+### Try C++ example
+
+Firstly you need to launch a milvus server.
+If you already build entire milvus project, just run:
```

Comment:
If you've already built the entire milvus project, just start Milvus server with the following command: